docs/UserGuide.md

```diff
@@ -400,12 +508,17 @@ Furthermore, certain edits can cause the InternBook to behave in unexpected ways
 **A**: For MAC users, open up your **Terminal**, and for Windows users, open up your **Command Prompt**.
 Type in `java -version` and you will see the java version.
 
+**Q**: How do I navigate to my folder in my terminal? <br>
+**A**: For **Windows** user:
+* Open Windows File Explorer.
+* Navigate to the folder that consist 
```

Comment:
I think this one you will have to update with the details
1. Open Windows File Explorer.
2. Navigate to the parent folder of the target folder and double-click into it (the folder you want to open a command window into)
3. Click on the file explorer address bar of the folder you are in and clear the File Path (Eg. C:\Users\PC\Downloads\internBook) and type “cmd” and press enter.

Mac
1. Open Finder.
2. Navigate to the parent folder of the target folder but do not double-click into it (the folder you want to open a terminal window into)
3. Right-Click on the folder and hover over Services (At the bottom), Click New Terminal at Folder. 
    - If there no such option, you will have to go to your Settings 	> Keyboard > Keyboard Shortcuts.. > Services > Files
    and Folders > Tick New Terminal at Folder
4. Now try again
A Terminal at the folder location will pop up.